src/ResourceManager/Kusto/Commands.Kusto/Commands.Kusto.csproj

```diff
@@ -0,0 +1,120 @@
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="ArmManagementClient, Version=1.0.5.0, Culture=neutral, PublicKeyToken=829a00acde12d551, processorArchitecture=AMD64">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Kusto.Management.1.0.5-beta\lib\net452\ArmManagementClient.dll</HintPath>
+    </Reference>
+    <Reference Include="Hyak.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.11\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime.Azure, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.Azure.3.3.12\lib\net452\Microsoft.Rest.ClientRuntime.Azure.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Data.Linq" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Commands\GetAzureRmKustoCluster .cs" />
+    <Compile Include="Commands\NewAzureRmKustoCluster.cs" />
+    <Compile Include="Commands\RemoveAzureRmKustoCluster.cs" />
+    <Compile Include="Commands\TestAzureRmKustoClusterName.cs" />
+    <Compile Include="Commands\UpdateAzureRmKustoCluster.cs" />
+    <Compile Include="Models\KustoClient.cs" />
+    <Compile Include="Models\KustoCmdletBase.cs" />
+    <Compile Include="Models\PSKustoCluster.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+    <Compile Include="Utilities\KustoUtils.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Commands.Common.Authentication.Abstractions.csproj">
```

Comment:
@radennis these common projects that are being referenced here have been removed from this repository and are now referenced externally as NuGet packages in the `Common.Dependencies.targets` file (which also includes other NuGet references to common libraries, such as `Microsoft.Rest.ClientRuntime`, `Newtonsoft.Json`, and others).

You can remove all of these references from your `.csproj` file and import this `.targets` file. Use the [`Commands.Billing.csproj`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Billing/Commands.Billing/Commands.Billing.csproj) file as a reference for what this should look like (`.targets` file is being imported [here](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Billing/Commands.Billing/Commands.Billing.csproj#L3))

Reply:
Sure I will do that, thanks!
I just followed this guide: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-developer-guide.md.

Was I supposed to follow a different guide?

Reply:
@elirazcc That is the correct guide. We are currently in a transitional phase, so our guide currently does not outline our common code changes we've done in the last couple months. We will be entirely NetStandard by end-of-year, so we'll be rewriting the guide by then. Basically, it wasn't worth the effort currently to write a guide for a temporary state of the repository. My apologies for the inconvenience.